python/lsst/analysis/tools/actions/plot/colorColorFitPlot.py

```diff
@@ -155,11 +155,11 @@ def makePlot(
           * Statistics that are shown on the plot or used by the plotting code:
              * ``approxMagDepth``
                    The approximate magnitude corresponding to the SN cut used.
-             * ``f"{self.plotName}_sigmaMAD"``
+             * ``f"{self.plotName}_nanmedian"``
                    The sigma mad of the distances to the line fit.
```

Comment:
Why has this changed from sigmaMad to median?

Reply:
Whoops, that was a find/replace fail I missed correcting.